tests/Unit/TestCase.php

```diff
@@ -61,5 +61,10 @@ protected function mockCommonWpFunctions() {
 		foreach ( $functions as $function ) {
 			Monkey\Functions\when( $function )->echoArg();
 		}
+
+		Monkey\Functions\when( 'is_wp_error' )
+			->alias( function() {
+				return $thing instanceof WP_Error;
```

Comment:
`WP_Error` is not loaded into memory in a unit test, as WordPress is not running. You'll need a mock or fixture here.

Reply:
Isn't what I'm doing here?